src/main/java/uk/gov/hmcts/dts/fact/services/CourtService.java

```diff
@@ -36,20 +41,25 @@
 
     private final MapitService mapitService;
     private final CourtRepository courtRepository;
+    private final ProximitySearch proximitySearch;
     private final CourtWithDistanceRepository courtWithDistanceRepository;
     private final ServiceAreaRepository serviceAreaRepository;
     private final ServiceAreaSearchFactory serviceAreaSearchFactory;
     private final FallbackProximitySearch fallbackProximitySearch;
 
+    private static final Logger LOGGER = LoggerFactory.getLogger(CourtService.class);
```

Comment:
We have already got projectlombok dependency in the FACT API, so can just use the class annotation @Slf4j without explicitly call getLogger here